sdk/mixedreality/Microsoft.Azure.Management.MixedReality/src/Microsoft.Azure.Management.MixedReality.csproj

```diff
@@ -6,7 +6,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.MixedReality</PackageId>
     <Description>Microsoft Azure Management Library of Mixed Reality cloud services.</Description>
-    <Version>0.9.0-preview</Version>
+    <Version>0.9.1-preview</Version>
```

Comment:
I'd recommend 0.9.0-preview.X which is the general versioning model that we are moving to. See https://github.com/Azure/azure-sdk/blob/master/docs/policies/releases.md#package-versioning.